package.json

```diff
@@ -16,15 +16,17 @@
     "node": ">=4.2.3"
   },
   "scripts": {
-	"compile": "babel -sd lib/ src/",
-	"prepublish": "npm run compile",
-	"test": "make test"
+    "compile": "babel -sd lib/ src/",
+    "prepublish": "npm run compile",
+    "test": "mocha --require babel-core/register --reporter spec test/test.jsx"
```

Comment:
I think you want to be using the [`babel-register`](https://babeljs.io/docs/usage/require/) package, not `babel-core/register`.